.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: java
+dist: trusty
```

Comment:
Had to force trusty since the packages specified below do not exist on more recent releases. Probably needs another ticket to upgrade the integration to use xenial (default) or bionic.